### ISSUE TITLE
enum field is defined in both scalars and the extended model with relations

### ISSUE THREAD

**Issue by capaj** [YassinEldeeb/prisma-tgql-types-gen#43]
## Bug description
If you have a field with an enum for type, it is generated in two places
![image](https://user-images.githubusercontent.com/1305378/153851038-db865870-ddd4-461d-93c3-802bb7612638.png)

## To Reproduce
generate a model with an enum using a config which has this:
```
  splitScalarAndObjectTypeFields = true
```
## Expected behavior
it is generated only in the scalars class

## Environment & setup
all versions

## Prisma  Version
```
Environment variables loaded from .env
prisma                  : 3.9.2
@prisma/client          : 3.9.2
Current platform        : debian-openssl-1.1.x
Query Engine (Node-API) : libquery-engine bcc2ff906db47790ee902e7bbc76d7ffb1893009 (at ../node_modules/prisma/node_modules/@prisma/engines/libquery_engine-debian-openssl-1.1.x.so.node)
Migration Engine        : migration-engine-cli bcc2ff906db47790ee902e7bbc76d7ffb1893009 (at ../node_modules/prisma/node_modules/@prisma/engines/migration-engine-debian-openssl-1.1.x)
Introspection Engine    : introspection-core bcc2ff906db47790ee902e7bbc76d7ffb1893009 (at ../node_modules/prisma/node_modules/@prisma/engines/introspection-engine-debian-openssl-1.1.x)
Format Binary           : prisma-fmt bcc2ff906db47790ee902e7bbc76d7ffb1893009 (at ../node_modules/prisma/node_modules/@prisma/engines/prisma-fmt-debian-openssl-1.1.x)
Default Engines Hash    : bcc2ff906db47790ee902e7bbc76d7ffb1893009
Studio                  : 0.457.0
Preview Features        : referentialIntegrity
```

**Comment by YassinEldeeb** [comment]
Fixed! just need to grab a new npm access token 😅

![Screenshot 2022-02-14 135500](https://user-images.githubusercontent.com/72823042/153859813-ec40b198-2e74-4e96-b09a-ea4b7bdc417e.png)

**Comment by YassinEldeeb** [comment]
Fixed in `v1.5.12` 🥳
